src/components/Slider/Slider.jsx

```diff
@@ -0,0 +1,242 @@
+import React, { forwardRef, useRef } from "react";
+import PropTypes from "prop-types";
+import { COLORS_BASIC, SIZES_BASIC } from "../../constants";
+import useMergeRefs from "../../hooks/useMergeRefs";
+import "./Slider.scss";
+import { bem } from "./SliderCommons";
+import { ensureValueDefault } from "./SliderHelpers";
+import { SliderProvider } from "./SliderContext";
+import SliderBase from "./SliderBase/SliderBase";
+import SliderInfix from "./SliderInfix";
+import { createBemBlockHelper } from "../../helpers/bem-helper";
+
+const Slider = forwardRef(
+  (
+    {
+      ariaLabel,
+      ariaLabeledBy,
+      className,
+      classNameBase,
+      color,
+      "data-testid": dataTestId,
+      disabled,
+      id,
+      isRange,
+      max,
+      min,
+      onChange,
+      step,
+      showValue,
+      size,
+      value,
+      valueDefault,
+      valueFormatter,
+      valueText,
+      // ------ Additional subcomponents' props
+      indicateSelection,
+      prefix,
+      postfix
+    },
+    ref
+  ) => {
+    const componentRef = useRef(null);
+    const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
+    const consumerBem = createBemBlockHelper(classNameBase, { isConsume: true });
+    const providerProps = {
+      ariaLabel,
+      ariaLabeledBy,
+      classNameBase,
+      color,
+      dataTestId,
+      disabled,
+      isRange,
+      max,
+      min,
+      onChange,
+      showValue,
+      size,
+      step,
+      value,
+      valueDefault: ensureValueDefault({ valueDefault, isRange, min, max }),
+      valueFormatter,
+      valueText,
+      infixOptions: {
+        prefix,
+        postfix,
+        indicateSelection
+      }
+    };
+    return (
+      <SliderProvider {...providerProps}>
+        <div
+          className={bem("", { disabled, "value-shown": showValue }, className)}
```

Comment:
I kind of not understand what is the new bem helper and how it is different from the old one. But because now we move to use CSS Modules I am not sure it is still necessary